jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/scrollspy/ScrollSpyWithNestedNavTests.java

```diff
@@ -198,29 +270,17 @@ public void linkClickableFocusTests(int index) {
 
 
     @Test(dataProvider = "focusCheck")
-    public void highlightFocusedItemsTests(int itemToClick,
-                                           int index_1, String class_1,
-                                           int index_2, String class_2,
-                                           int index_3, String class_3,
-                                           int index_4, String class_4,
-                                           int index_5, String class_5,
-                                           int index_6, String class_6,
-                                           int index_7, String class_7) {
-        List<Integer> elementsIndex = Arrays.asList(index_1, index_2, index_3, index_4, index_5, index_6, index_7);
-        List<String> elementsClass = Arrays.asList(class_1, class_2, class_3, class_4, class_5, class_6, class_7);
-        int classIndex = 0;
+    public void highlightFocusedItemsTests(int itemToClick, Map<Integer, String> itemsMap) {
 
         nestedNav.navItemLink.get(itemToClick).highlight();
         nestedNav.navItemLink.get(itemToClick).click();
 
-        for (int elIndex : elementsIndex) {
-            nestedNav.navItemLink.get(elIndex).is()
-                    .core()
-                    .displayed()
-                    .enabled()
-                    .cssClass(elementsClass.get(classIndex));
-            classIndex++;
-        }
+        itemsMap.keySet().forEach(key ->
+                nestedNav.navItemLink.get(key).is()
+                        .core()
+                        .displayed()
+                        .enabled()
+                        .cssClass(itemsMap.get(key)));
```

Comment:
We could use here hasClass() method, checking only single element that it has class 'active'. So we could get rid of full strings of classes and big maps